code/workspaces/web-app/src/reducers/stacksReducer.js

```diff
@@ -17,23 +18,23 @@ const initialState = {
 
 export default typeToReducer({
   [LOAD_STACKS_ACTION]: {
-    [PROMISE_TYPE_PENDING]: () => ({ ...initialState, fetching: true }),
+    [PROMISE_TYPE_PENDING]: state => ({ ...initialState, value: [...state.value], fetching: true }),
```

Comment:
Does the `value` need to be created as a new array? I know the overall returned object of the reducer needs to be a new object for redux to work properly, but don't know whether it is the same for the nested variables. Not sure if it makes much of a difference either way but might be a bit more performant if that ever becomes an issue.

Reply:
Fair point, value isn't being mutated so doesn't need spreading.